tasks/bumppom.js

```diff
@@ -90,6 +90,15 @@ module.exports = function(grunt) {
 
     var _total = src.length;
     var _backup_dest = '';
+    
+    var fnReplaceSnapshot = function(version){
+      return (options.use_snapshot) ? version.replace(/\-.*$/, '-SNAPSHOT') : version;
```

Comment:
I know what you mean and probably I should append it like in "1.2.3-0-SNAPSHOT." because the prelease version information gets actually lost.

But to be honest I don't use different prelease versions nor different Snapshots because I use them only for continuous integration and these snapshots always represent the latest git pushes.

When different snapshots/development-versions are really needed, they are handled through git branches.

(I simply have no good use case for a multiple snapshot usage yet I could refer to)
Any prerelease of 1.2.3-snapshot will be deployed to a sonatype nexus maven for continuous integration (doing *mvn clean* builds) in other snapshot maven projects

So this pull request probably represents *my use case only*, but I wanted to keep it **simple until** there are new use cases or **suggestions** for further configuration :)
For now I would only need *this use case*